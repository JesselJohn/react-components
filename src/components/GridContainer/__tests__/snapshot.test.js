import React from "react";
import { shallow, mount } from "enzyme";
import { shallowToJson } from "enzyme-to-json";

import TestComponent from "../test.component";

describe("Root Component Tests", () => {
  it("Matches Shallow SnapShot", done => {
    expect(shallowToJson(shallow(<TestComponent />))).toMatchSnapshot();
    done();
  });

  it("Matches Mounted SnapShot", done => {
    const mounted = mount(<TestComponent />);

    setTimeout(() => {
      expect(mounted).toMatchSnapshot();
      done();
    }, 300);
  });
});
