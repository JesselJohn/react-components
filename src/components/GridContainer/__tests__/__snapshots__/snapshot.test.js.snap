// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Root Component Tests Matches Mounted SnapShot 1`] = `
<GridContainerTest>
  <GridContainer
    className=""
    strict={false}
  >
    <div
      className="nwc-grid-container  "
    >
      <div>
        Hello
      </div>
    </div>
  </GridContainer>
</GridContainerTest>
`;

exports[`Root Component Tests Matches Shallow SnapShot 1`] = `
<GridContainer
  className=""
  strict={false}
>
  <div>
    Hello
  </div>
</GridContainer>
`;
